### PR TITLE
Unity 2019.4.30f1

### PR DESCRIPTION
**Pull request by bruno-garcia**
Added a changelog entry since this could affect how building/testing and packaging is done.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
             unity-installation-path: /Applications/Unity/
             unity-root: /Applications/Unity/Hub/Editor/
             unity-path: Unity.app/Contents/MacOS
-          - unity-version: 2019.4.21f1
+          - unity-version: 2019.4.30f1
```
Comment (bruno-garcia) [inline .github/workflows/ci.yml:34]: Do we really need this here if we have it in the matrix?

File: samples/unity-of-bugs/Packages/manifest.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "dependencies": {
-    "com.unity.ide.rider": "1.1.4",
-    "com.unity.test-framework": "1.1.24",
+    "com.unity.ide.rider": "1.2.1",
+    "com.unity.ide.visualstudio": "2.0.11",
```
Comment (SimonCropp) [inline samples/unity-of-bugs/Packages/manifest.json:4]: should updating unity also add an extra entry `com.unity.ide.visualstudio`?

**Review by SimonCropp — APPROVED** [review]

File: samples/unity-of-bugs/Packages/manifest.json (hunk 2 above)
Comment (bitsandfoxes) [reply samples/unity-of-bugs/Packages/manifest.json:4]: Fair point! Unity-Package cleanup: https://github.com/getsentry/sentry-unity/pull/351 (has been on my "want to do" for ages!)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bitsandfoxes) [reply .github/workflows/ci.yml:34]: We intend to run CI with multiple Unity versions: 2019, 2020, 2021, and I put it there so I can associate the version changeset with the version number for installing via the hub.
Like .NET does it [here](https://github.com/getsentry/sentry-dotnet/blob/465e9031c24d2574ded6c5f6e686eb9030c6e879/.github/workflows/build.yml#L19).

**Pull request merged**